src/main/java/uk/gov/hmcts/reform/civil/handler/callback/user/RespondToClaimCallbackHandler.java

```diff
@@ -475,16 +465,8 @@ && ofNullable(caseData.getRespondent2Copy()).isPresent()) {
                     .nextDeadline(applicant1Deadline.toLocalDate());
             }
             // if present, persist the 2nd respondent address in the same fashion as above, i.e ignore for 1v1
-            if (ofNullable(caseData.getRespondent2()).isPresent()
-                && ofNullable(caseData.getRespondent2Copy()).isPresent()) {
-                var updatedRespondent2 = caseData.getRespondent2().toBuilder()
-                    .primaryAddress(caseData.getRespondent2Copy().getPrimaryAddress())
-                    .build();
-
-                updatedCaseDataBuilder
-                    .respondent2(updatedRespondent2)
-                    .respondent2Copy(null)
-                    .respondent2DetailsForClaimDetailsTab(updatedRespondent2.toBuilder().flags(null).build());
+            if (ofNullable(caseData.getRespondent2()).isPresent()) {
+                PersistDataUtils.persistPartyAddress(oldCaseData, caseData);
```

Comment:
What about the respondent2DetailsForClaimDetailsTab? this part doesnt consider it right?